.gitignore

```diff
@@ -1,48 +1,49 @@
-
-*.ini
```

Comment:
It was a change from CRLF to LF. It wasn't my intention nor my knowledge that .gitignore change line endings but since all files are supposed to be LF I just ran with it.